lsmb-request.pl

```diff
@@ -19,6 +19,7 @@ =head1 COPYRIGHT
 =cut
 
 package LedgerSMB::Handler;
+use lib '.';
```

Comment:
Ok, thanks for the corrections and valid points.
As discussed with ehuelsmann it seems releasing with `use '.';` is the first step, then we can consider going further down the track.